synapse/federation/sender/__init__.py

```diff
@@ -62,12 +62,12 @@
 logger = logging.getLogger(__name__)
 
 sent_pdus_destination_dist_count = Counter(
-    "synapse_federation_client_sent_pdu_destinations:count",
+    "synapse_federation_client_sent_pdu_destinations_count",
     "Number of PDUs queued for sending to one or more destinations",
 )
 
 sent_pdus_destination_dist_total = Counter(
-    "synapse_federation_client_sent_pdu_destinations:total",
+    "synapse_federation_client_sent_pdu_destinations",
     "Total number of PDUs queued for sending across all destinations",
```

Comment:
> (N.B. I also fix a metric that I forgot to rename in https://github.com/matrix-org/synapse/pull/13540)

Any way to check if we've missed any others? I guess grep for `"synapse_.*:.*"`?

Reply:
Should have thought of that; suppose I never really twigged that they all have a convenient `synapse_` prefix that makes them greppable instead of 'string with colon' in :-)

Found another pair of sneaky ones ...

Reply:
>  I never really twigged that they all have a convenient synapse_ prefix

I'm just guessing here!!

Could also grep for something like `Counter\(\n?.*".*:` and `Gauge\(\n?.*".*:` if you can get the multiline regex stuff working

Reply:
Conventionally we definitely are meant to use an easily-recognised prefix like `synapse_`. I don't think even we would ignore that...